### ISSUE TITLE
PEFT

### ISSUE THREAD

**Issue by Kaisor-Yuan** [mshukor/UnIVAL#3]
In "unival_caption_stage_s2_onlylinear.sh", I cannot find the path the data_dir refer to. Is there any instruction on how to fine-tune on my own dataset?

**Comment by mshukor** [comment]
The scripts for PEFT training are the same as those for normal/all-parameters training ,except some changes to hyperparameters and freezing of the encoders/LM. 

For details about data preprocessing and format you can check [this repo"Prepare the Dataset & Checkpoints"](https://github.com/OFA-Sys/OFA/tree/main#image-captioning). We don't use object labels and you can avoid saving images as base64 string by rather providing the image path and the argument "--read-from-img-path".

For the `unival_caption_stage_s2_onlylinear.sh`, the data_dir contains the train (caption_stage1_train.tsv shuffled to caption_stage1_train_i.tsv) and val data.
